api/tests/unit/integrations/amplitude/test_unit_amplitude_models.py

```diff
@@ -0,0 +1,26 @@
+from django.db.models import Q
+
+from integrations.amplitude.models import AmplitudeConfiguration
+
+
+def test_amplitude_configuration_save_writes_environment_to_dynamodb(
```

Comment:
Think we should add test for delete as well?

Reply:
Done 👍